src/geometry/adjoints.jl

```diff
@@ -0,0 +1,58 @@
+function ChainRulesCore.rrule(::Interpolations.BSplineInterpolation, T, A, it, axs)
```

Comment:
Do you need an rrule for this method if you already have one for the type constructor? What happens if you remove it?

Reply:
It actually does not make a difference! I can do away with it to be honest, it always falls back to the constructor adjoints. Should I do that? I'll add it again if we collude with it at some latter part.